terraform/a2ha-terraform/modules/opensearch/templates/provision.sh.tpl

```diff
@@ -61,7 +61,7 @@ if [ -e /hab/sup/default/specs/"$OS_PKG_NAME".spec ]; then
      #appliedConfigFile="config.$RANDOM.toml"
      #automate-backend-ctl applied --svc=automate-ha-opensearch | tail -n +2 > $appliedConfigFile
 
-     bash -c 'sysctl -w vm.max_map_count=262144'
+     bash -c 'hab pkg exec core/procps-ng sysctl -w vm.max_map_count=262144 "$LOGCMD"'
```

Comment:
Check if the core/procps-ng is installed at the time of executing this command. 
please add screenshot of file were the output is populated in the logs

Reply:
Now the scope of this PR has changed